johnsonandjohnson/Bodiless-JS
# feat(layouts): Allow specifying default width for flexbox item.

## Changes
- Create a `defaultWidth` prop on the flexbox which specifies a desired default width for new items.

## Test Instructions
- Try various values of this prop on the /flexbox page and observe the width of new items.

## Related Issues
Fixes #162
